entity-framework/core/miscellaneous/testing/testing-sample.md

```diff
@@ -11,25 +11,26 @@ no-loc: [Item, Tag, Items, Tags, items, tags]
 
 > [!TIP]
 > The code in this document can be found on GitHub as a [runnable sample](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/Miscellaneous/Testing/ItemsWebApi/).
-> Note that some of these tests **are expected to fail**. The reasons for this are explained below. 
+> Note that some of these tests **are expected to fail**. The reasons for this are explained below.
 
 This doc walks through a sample for testing code that uses EF Core.
 
 ## The application
 
 The [sample](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/Miscellaneous/Testing/ItemsWebApi/) contains two projects:
-- ItemsWebApi: A very simple [Web API backed by ASP.NET Core](/aspnet/core/tutorials/first-web-api?view=aspnetcore-3.1&tabs=visual-studio) with a single controller
+
+- ItemsWebApi: A very simple [Web API backed by ASP.NET Core](/aspnet/core/tutorials/first-web-api) with a single controller
 - Tests: An [XUnit](https://xunit.net/) test project to test the controller
 
 ### The model and business rules
 
 The model backing this API has two entity types: Items and Tags.
 
-* Items have a case-sensitive name and a collection of Tags.
-* Each Tag has a label and a count representing the number of times it has been applied to the Item.
-* Each Item should only have one Tag with a given label.
-  * If an item is tagged with the same label more than once, then the count on the existing tag with that label is incremented instead of a new tag being created. 
-* Deleting an Item should delete all associated Tags.
+- Items have a case-sensitive name and a collection of Tags.
```

Comment:
Do we have an official standard for list items??

Reply:
I think we should decide one. vscode ask it to be same in the file.

Reply:
Filed #2624